src/scene/index.js

```diff
@@ -53,22 +54,26 @@ export function getCamera() {
   return camera;
 }
 
+export function isDragging() {
```

Comment:
This function `isDragging` is currently unused. I added it so that if another js file needed to know if the Globe was being "dragged", then it's already exposed, in a similar manner to `isHovering` (which is used elsewhere).

I can remove this unused function if we'd prefer 🤷 I think of it like an "API" method for our code.